src/status_im/chat/deprecated_subs.cljs

```diff
@@ -0,0 +1,84 @@
+(ns status-im.chat.deprecated-subs
```

Comment:
Trying to follow along here. Is the current situation that the new chat.db and contact.db modifications will allow for new features (such as blocking) that were not possible before? Yet migrating all of the app over to use those would be such a massive undertaking that it is outside the scope of any one PR and therefore this namespace, *deprecated-subs* is needed?

Reply:
yes the new subs are computing the active chats and all metadata associated dynamically, including the list of ordered messages, datemarks and metadata. but to avoid 500 extra lines of code I move the subs old subs to deprecated-subs namespace and will update the views in future updates.

Reply:
👍 thanks for clarifying.